src/components/LocationPage/Recommendations.tsx

```diff
@@ -38,8 +39,10 @@ const Recommendations = ({
     alarmLevel,
   );
 
+  const isMobile = useBreakpoint(600);
+
   const containerProps = {
-    collapsedHeightMobile: '275px',
+    collapsedHeightMobile: '350px',
     collapsedHeightDesktop: '165px',
```

Comment:
It's tempting to rework the component so that you can specify `undefined` for `collapsedHeightDesktop` and then the component would only collapse/expand on mobile.  I think that'd be a bit more elegant (we don't have to pass this `collpasedHeightDesktop` property that is irrelevant and the `<Recommend ... />` code wouldn't need to be duplicated below).  It'd also be more reusable (if there are other places that we want expand/collapse only on mobile then we could use this same mechanism).

But what you have is okay and we could refactor if/when we have another usage for this functionality. 🤷

Reply:
I agree :) I'll add a TODO in case we have another usage.